ui/app/templates/vault/cluster/secrets/backend/list.hbs

```diff
@@ -45,27 +45,46 @@
         {{/if}}
       </div>
     {{/each}}
+    {{#if (gt model.meta.lastPage 1) }}
+      {{list-pagination
+        page=model.meta.currentPage
+        lastPage=model.meta.lastPage
+        link=(concat "vault.cluster.secrets.backend.list" (if (not baseKey.id) "-root"))
+        model=(compact (array backend (if baseKey.id baseKey.id)))
+      }}
+    {{/if}}
   {{else}}
-    <BlockEmpty>
-      {{#if (eq baseKey.id '')}}
-        There are currently no {{pluralize options.item}} in this backend.
-      {{else}}
-        {{#if filterIsFolder}}
-            {{#if (eq filter baseKey.id)}}
-              There are no {{pluralize options.item}} under <code>{{or filter}}</code>.
-            {{else}}
-              We couldn't find a folder matching <code>{{filter}}</code>.
-            {{/if}}
+    <div class="empty-state">
```

Comment:
I think this is another place we can push some of the computing to the controller (or could do with the `let` helper in the template) and use `EmptyState` directly.